### PR TITLE
Use nicer device names more often.

### PR DESCRIPTION
**Pull request by floitsch**
Instead of just printing the device-id, print its name.

### CODE REVIEW

File: src/cli/broker.toit [hunk 1]
```diff
@@ -527,7 +534,8 @@ class Broker:
     upgrade-from := []
     if known-encoded-firmwares.is-empty:
       if base-firmwares.is-empty:
-        ui_.warning "Firmware of device '$device-id' is unknown. Upgrade might not use patches."
+        short := device-short-strings_[device-id]
```
Comment (kasperl) [inline src/cli/broker.toit:537]: Maybe abstract getting the name of a device out, so you have one place to explain that device-short-strings_ must be set -- or check for null?

File: src/cli/broker.toit [hunk 2]
```diff
@@ -649,7 +658,8 @@ class Broker:
   device-for --id/uuid.Uuid -> DeviceDetailed:
     devices := broker-connection_.get-devices --device-ids=[id]
     if devices.is-empty:
-      ui_.abort "Device $id not found."
+      short := device-short-strings_[id]
+      ui_.abort "Device $short not found."
```
Comment (kasperl) [inline src/cli/broker.toit:662]: Not found sounds a bit like our implementation is wrong. If the user specified the wrong device name or id, maybe we should tell them that it is unknown or non-existing?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/cli/broker.toit (hunk 1 above)
Comment (floitsch) [reply src/cli/broker.toit]: done.

File: src/cli/broker.toit (hunk 2 above)
Comment (floitsch) [reply src/cli/broker.toit]: done.

**Pull request merged**